### ISSUE TITLE
Enable some privilege 

### ISSUE THREAD

**Issue by trotyoung** [thebitguru/alfred-imessage-2fa-copy#2]
By default, this workflow was not able to fetch code from the message, you just need to set something in the `system settings` -> `Privacy & Security`, give `Full Disk Access` and `Accessibility`. Then, it should be able to work. Thank you! This is a wonderful project.

**Comment by thebitguru** [comment]
Thanks for reporting! I will update the documentation accordingly and see if there is a way to check.

Any chance you can share a screenshot of the Settings screen?

**Comment by trotyoung** [comment]
Sure, these are my settings. Hope these help.
![Xnip2023-07-05_22-22-09](https://github.com/thebitguru/alfred-simple-2fa-paste/assets/19234895/48c89d64-4a77-4af4-930e-5c120a1413ff)
![Xnip2023-07-05_22-21-35](https://github.com/thebitguru/alfred-simple-2fa-paste/assets/19234895/005e7e77-231b-4373-aede-1a57b36a694a)
![Xnip2023-07-05_22-20-59](https://github.com/thebitguru/alfred-simple-2fa-paste/assets/19234895/203f63f1-06c4-4445-a833-b3949ea82f0e)

**Comment by thebitguru** [comment]
I have added this check in the [latest release](https://github.com/thebitguru/alfred-simple-2fa-paste/releases/tag/v1.0). Thanks for reporting it!
